utils/cliutils/commandsflags.go

```diff
@@ -2091,6 +2096,9 @@ var commandFlags = map[string][]string{
 	SyncStatus: {
 		branch, repository, serverId,
 	},
+	Setup: {
+		serverId, url, user, password, accessToken, Project, setupRepo,
```

Comment:
Let's add here the entire Jamaa, like in other commands
```suggestion
		serverId, url, user, password, accessToken, sshPassphrase, sshKeyPath, ClientCertPath, ClientCertKeyPath, Project, setupRepo,
```